.github/workflows/ci.yaml

```diff
@@ -204,14 +204,21 @@ jobs:
     runs-on: ubuntu-latest
     timeout-minutes: 5
 
+    strategy:
+      fail-fast: false
+      matrix:
+        php-version:
+          - "8.4"
+          - "8.5"
```

Comment:
was not sure how to configure CI so it runs the new test with opcache+coverage-driver.